.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,368 +11,7 @@ Thank you for your contribution! Follow this checklist to help us incorporate yo
-        </a><br>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/badge/icon?subject=Lint">
-        </a><br>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/badge/icon?subject=Docker">
-        </a><br>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/badge/icon?subject=Docs">
-        </a>
-      </td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-    </tr>
-    <tr>
-      <td>Portable</td>
-      <td>---</td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-      <td>
-        <a href="https://ci-beam.apache.org/job/beam_PreCommit_GoPortable_Cron/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PreCommit_GoPortable_Cron/lastCompletedBuild/badge/icon">
-        </a>
-      </td>
-      <td>---</td>
-      <td>---</td>
-      <td>---</td>
-    </tr>
-  </tbody>
-</table>
-
-See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.
-
+To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/validate-runner/README.md)
```

Comment:
There is a typo in the link. We can also expand this:
```suggestion
If some tests don't pass on you PR, investigate the errors by clicking on `Details` link. To rule out errors external to your PR, check the [build health dashboard](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md). You can search https://s.apache.org/beam-flakes for known test flakes.
```

Reply:
https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md isn't a valid link either.